aztec/src/main/kotlin/org/wordpress/aztec/toolbar/AztecToolbar.kt

```diff
@@ -505,7 +505,7 @@ class AztecToolbar : FrameLayout, IAztecToolbar, OnMenuItemClickListener {
             if (action != ToolbarAction.ELLIPSIS_COLLAPSE &&
                     action != ToolbarAction.ELLIPSIS_EXPAND) {
                 val view = findViewById<ToggleButton>(action.buttonId)
-                if (view.isChecked) actions.add(action)
+                if (view?.isChecked == true) actions.add(action)
```

Comment:
Hi @cameronvoell, this change should fix the `IllegalStateException` failures on CI. At first I debated whether this was the right solution, but I think it makes sense. Before this change, we were guaranteed that all toolbar actions were always present. But now that background color is an optional plugin, for the first time ever it's possible that one of the actions in `ToolbarAction.values()` is not present.

An alternative way to fix these tests would've been to move the following out of the `if (!isRunningTest)` branch:
```kotlin
            aztec.addPlugin(CssBackgroundColorPlugin())
            aztec.addPlugin(BackgroundColorButton(visualEditor))
```

TBH I'm not exactly sure why we chose to put that inside `if (!isRunningTest)` in the first place, but I felt that this was a better solution overall to prevent crashes from users of the library that don't add the plugin.

Unfortunately this fix alone does not resolve all of the espresso tests though. Some of them are fixed, but some of them now have different failures. One such example is `FormattingHistoryTests.testAddBoldUndoRedo()`, which now fails like this:
```
Expected: EditText matches <strong>Testing</strong>
     Got: "SourceViewEditText{ ... text=<strong><b>T</b></strong><b>esting</b> ... }"
```

I have no idea why the changes in this PR would cause the `<b>` tags to be added, or why they'd be cutting of the "T" in "Testing". It maybe due to changes that occurred before I got involved. As someone with more context on this project, do you have any ideas what might be causing this?

Thanks very much!

Reply:
I'll take a look.